diff -urN mpv-0.4.0.orig/wscript_build.py mpv-0.4.0/wscript_build.py
--- mpv-0.4.0.orig/wscript_build.py	2014-06-28 12:59:05.435131074 +0200
+++ mpv-0.4.0/wscript_build.py	2014-06-28 12:59:38.128463410 +0200
@@ -539,7 +539,7 @@
     if ctx.dependency_satisfied('zsh-comp'):
         ctx.zshcomp(target = "etc/_mpv")
         ctx.install_files(
-            ctx.env.DATADIR + '/zsh/vendor-completions',
+            ctx.env.DATADIR + '/zsh/site-functions',
             ['etc/_mpv'])
 
     ctx.install_files(
