--- mpv-0.5.0.orig/wscript_build.py	2014-08-12 18:58:12.336633168 +0200
+++ mpv-0.5.0/wscript_build.py	2014-08-12 19:05:20.233234905 +0200
@@ -451,7 +451,9 @@
     build_static = ctx.dependency_satisfied('libmpv-static')
     if build_shared or build_static:
         if build_shared:
-            ctx.load("syms")
+            import os
+            waftoolsdir = os.path.dirname(__file__) + os.sep + "waftools"
+            ctx.load("syms", tooldir=waftoolsdir)
         vre = '^#define MPV_CLIENT_API_VERSION MPV_MAKE_VERSION\((.*), (.*)\)$'
         libmpv_header = ctx.path.find_node("libmpv/client.h").read()
         major, minor = re.search(vre, libmpv_header, re.M).groups()
